assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -2196,7 +2355,14 @@ tests:
               relation: viewer
             expectation:
               - document:public
-
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              object: document:public
+              relation: viewer
+            expectation:
+              - user:*
```

Comment:
Would you please add a tuple `user:jon` above and assert we return the wildcard along with the explicit user as well? e.g.

```
expectation:
  - user:*
  - user:jon
```